extensions/healthie/actions/createMetricEntry.ts

```diff
@@ -0,0 +1,111 @@
+import { isNil } from 'lodash'
+import { HealthieError, mapHealthieToActivityError } from '../errors'
+import {
+  FieldType,
+  type Action,
+  type Field,
+} from '@awell-health/extensions-core'
+import { Category } from '@awell-health/extensions-core'
+import { getSdk } from '../gql/sdk'
+import { initialiseClient } from '../graphqlClient'
+import { type settings } from '../settings'
+
+const fields = {
+  userId: {
+    id: 'userId',
+    label: 'User ID',
+    description: 'The ID of the patient that this entry should be attached to.',
+    type: FieldType.STRING,
+    required: true,
+  },
+  category: {
+    id: 'category',
+    label: 'Category',
+    description: 'Specifies what kind of metric we are storing.',
+    type: FieldType.STRING,
+    required: true,
+  },
+  metricStat: {
+    id: 'metricStat',
+    label: 'Metric stat',
+    description:
+      'This is the actual data value for the metric. e.g if a patient weights 182 lbs, you would pass in 182 with a category of Weight',
+    type: FieldType.STRING,
```

Comment:
Metrics are inherently numbers so a numeric type would fit better here. Just make sure we stringify it before sending it over to Healthie's API.